policyengine_uk/parameters/gov/hmrc/pensions/pension_contributions_relief_age_limit.yaml

```diff
@@ -0,0 +1,10 @@
+description: The pensions contributions relief is limited to filers below this age threshold.
+values:
+  2004-01-01: 75
+metadata:
+  unit: year
+  label: pension contributions relief age limit
+  period: year
+  reference:
+    - title: "Finance Act 2004, s. 188"
```

Comment:
generally we try to cite the legal sources as directly as possible to show what section of the legal code is cited 
e.g. here it would be:
```
- title: Finance Act 2004, s. 188 (3)(a)
```

Reply:
this helps us with making sure that we are as confident as possible in the source and that we can make future adjustments as needed

Reply:
all parameter files